domains/mi-dominio.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "guardini-philama",
```

username doesn't match github username
```suggestion
    "username": "Philama",
```